src/app/components/ImageWithCaption/index.tsx

```diff
@@ -40,13 +40,15 @@ const renderCaption = (block: object, type: string, service: Services) =>
 
 type Props = {
   blocks: object[];
+  className?: string;
```

Comment:
Do we need to pass `className` down to here and the `LivePageMediaPlayer`? I can see they're being set to `mediaStyles` but can't see where `mediaStyles` is declared.

Reply:
Not sure I can answer this, but I can clarify where this logic comes from in case it helps.

I suggested it since it allows you to a) [apply styles](https://emotion.sh/docs/css-prop#use-the-css-prop) to Legacy components that don't take the css prop, and b) [override styles](https://emotion.sh/docs/css-prop#style-precedence). We used it on the MediaAriclePage to pass css styles down to nested legacy components.

This method works here - but there might be a better approach to take.